meta/saimetadatatypes.h

```diff
@@ -190,11 +190,6 @@ typedef enum _sai_attr_value_type_t
     SAI_ATTR_VALUE_TYPE_INT32_RANGE,
 
     /**
-     * @brief Attribute value is vlan list.
-     */
-    SAI_ATTR_VALUE_TYPE_VLAN_LIST,
-
-    /**
```

Comment:
please do not shuffle sort or reorder already existing enums since you are changin their numeric values and lib is not backward compatible in that way